test/components/converters/test_azure_ocr_doc_converter.py

```diff
@@ -10,7 +10,7 @@
 class TestAzureOCRDocumentConverter:
     def test_init_fail_wo_api_key(self, monkeypatch):
         monkeypatch.delenv("AZURE_AI_API_KEY", raising=False)
-        with pytest.raises(ValueError, match="AzureOCRDocumentConverter expects an Azure Credential key"):
+        with pytest.raises(ValueError):
```

Comment:
The error message changed, but I think we don't want to rely on error messages anymore.
So I only check the error type.